app/lectures/models.py

```diff
@@ -0,0 +1,16 @@
+from django.contrib.auth import get_user_model
+from django.db import models
+from subjects.models import Subject
+
+
+# Create your models here.
+
+
+class Lecture(models.Model):
+    title = models.CharField(max_length=100, null=True, blank=True)
+    text = models.TextField()
+    user = models.ForeignKey(get_user_model(), on_delete=models.CASCADE)
+    subject = models.ForeignKey(Subject, on_delete=models.CASCADE)
```

Comment:
`on_delete=models.CASCADE` will delete lecture object as well as referenced subject (and user for user field). I guess it not what expected. `on_delete=models.PROTECT` would be a better choice here.